systemd-units/plymouth-poweroff.sh.in

```diff
@@ -2,4 +2,5 @@
 
 if [ "$1" = "poweroff" ]; then
 	@PLYMOUTH_CLIENT_DIR@/plymouth quit --retain-splash
+	sleep 1
```

Comment:
Do we really care since we are powering off? My idea is that for some reason systemd is killing the process too quickly